src/index.js

```diff
@@ -12,6 +12,23 @@ function findTagInParents(element, tagName) {
   return findTagInParents(element.parentNode, tagName);
 }
 
+let sortListByTabIndex;
+let nodeVersion = +process.version.slice(1).split('.')[0];
+if (nodeVersion < 11) {
```

Comment:
Is there any way we can make this consistent regardless of node version? I'm not super thrilled about the added complexity here 😬

Reply:
Yeah I thought about that but it would mean an additional map over the entire list.  Felt like it would be punishing node 12 for the sins of node 10.  Also didn't think it warranted adding a separate sort just because of this. As always open to other options I might be missing

Reply:
I don't think we should be worried about perf at this stage... We're paying the price of complexity for an unnoticeable perf gain 🤷‍♂️

Reply:
alrighty.  pushed a version that just has a single sort @kentcdodds